source/StarField.cpp

```diff
@@ -82,6 +87,19 @@ void StarField::SetHaze(const Sprite *sprite)
 	if(!sprite)
 		sprite = SpriteSet::Get("_menu/haze");
 	
+	if(isFading){
+		return;
+	}
```

Comment:
At first I used isFading to delay the sprite change because additional call during fading may cause sharp change of the haze. (Perhaps it rarely happens as the fade effect is short in time.)

But I checked again that, as you mentioned, isFading only blocks the additional call and doesn't delay the sprite change. So I will remove the isFading first.